mb_ALL-LINKS.user.js

```diff
@@ -112,6 +112,227 @@ var whitelistSearchLinks = {
+							"https://www.metal-archives.com/search?searchString=%label-name%&type=label_name",
+							"https://www.metal-archives.com/search?searchString=%release-group-name%&type=album_title",
+							"https://www.metal-archives.com/search?searchString=%release-name%&type=album_title"
+						],
+						"Metal-Archives (aliases)": "https://www.metal-archives.com/search?searchString=%artist-name%&type=artist_alias",
+						"Metal-Archives (bands)": "https://www.metal-archives.com/search?searchString=%artist-name%&type=band_name",
+						"Spirit of Metal": [
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/labels.php?l=%language%&search=%label-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/places.php?l=%language%&search=%place-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/albums.php?l=%language%&search=%release-group-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/albums.php?l=%language%&search=%release-name%"
+							},
+						],
+						"Spirit of Metal (bands)": {
+							"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/bands.php?l=%language%&search=%artist-name%"
+						},
+						"Spirit of Metal (other artists)": {
+							"cn de en es fr pl pt ru": "https://www.spirit-of-metal.com/artists.php?l=%language%&search=%artist-name%"
+						}
+					}
+				},
+				rock: {
+					title: "Rock",
+					items: {
+						"Prog Archives": [
+							"http://www.progarchives.com/google-search-results.asp?cx=artists&q=%artist-name%",
+							"http://www.progarchives.com/google-search-results.asp?cx=artists&q=%release-group-name%"
+						],
+						"Prog Archives (by barcode)": "http://www.progarchives.com/google-search-results.asp?cx=artists&q=%release-barcode%",
+						"Prog Archives (by name)": "http://www.progarchives.com/google-search-results.asp?cx=artists&q=%release-name%",
+						"Spirit of Rock": [
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/labels.php?l=%language%&search=%label-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/places.php?l=%language%&search=%place-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/albums.php?l=%language%&search=%release-group-name%"
+							},
+							{
+								"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/albums.php?l=%language%&search=%release-name%"
+							},
+						],
+						"Spirit of Rock (bands)": {
+							"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/bands.php?l=%language%&search=%artist-name%"
+						},
+						"Spirit of Rock (other artists)": {
+							"cn de en es fr pl pt ru": "https://www.spirit-of-rock.com/artists.php?l=%language%&search=%artist-name%"
+						}
+					}
+				},
+				soundtrack: {
+					title: {
+						de: "Audiospur",
+						en: "Soundtrack",
```

Comment:
```suggestion
						"de en": "Soundtrack",
```
"Audiospur" is a term which is used in context of audio and video tracks, e.g. in editing software.

Reply:
Thanks, applied in commit https://github.com/jesus2099/konami-command/pull/756/commits/f0bf1cfe2109d5bd3d16ee4661cfcca32ceca682.